static/css/timetable/framework/page_layout.scss

```diff
@@ -10,7 +10,6 @@ h5,
 h6,
 p,
 a,
-span,
```

Comment:
For some reason, all span elements were set to [this](https://github.com/jhuopensource/semesterly/blob/52605bb9531f83547947ebc4853e84f9015a23a9/static/css/timetable/base/themes.scss#L83) blue color. This seemed